### ISSUE TITLE
Database of Patient information is empty

### ISSUE THREAD

**Issue by Masswear** [zdavatz/aips2sqlite#69]
When trying to generate a database for the patient information with java -jar aips2sqlite.jar --pinfo --lang=de --verbose --reports, the generated database is empty, despite showing that 4499 PI were processed and the report file containing all the prearations.

**Comment by zdavatz** [comment]
which Java Version?

**Comment by Masswear** [comment]
Java SE 23.0.1 (latest release)

**Comment by zdavatz** [comment]
Did you try to increase the Heap size? 

`-Xmx8048m`

**Comment by Masswear** [comment]
Yes, increased Heap size to 8048 MB, which works for Fachinfo, but not for Patientinfo. I don’t get an out of memory error, the program finishes successfully and creates an empty database.I run it in the same directory where I already have the output for Fachinfo, could this be the problem?Am 18.12.2024 um 03:02 schrieb Zeno R.R. Davatz ***@***.***>:﻿
Did you try to increase the Heap size?
-Xmx8048m

**Comment by zdavatz** [comment]
What is the Full command you are using?

**Comment by Masswear** [comment]
java -jar -Xmx8048m aips2sqlite.jar --lang=de --pinfo --verbose --reports

**Comment by zdavatz** [comment]
Yes, correct, I can confirm this. Will look into this.

**Comment by zdavatz** [comment]
Looks like this feature is not yet supported. Do you want us to support this feature?
